openapis/swagger.yaml

```diff
@@ -137,6 +139,12 @@ paths:
       description: ''
       operationId: listEntries
       parameters:
+        - name: form
+          in: query
+          description: specify wheather you want to list bundle, objects or both. 
```

Comment:
nit: whether 
Also what's the default? Is it just all?

Reply:
it defaults to object for the `index` endpoint